x-pack/test/security_solution_cypress/serverless_config.ts

```diff
@@ -36,6 +36,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ])}`,
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'disableTimelineSaveTour',
+          'riskEnginePrivilegesRouteEnabled',
         ])}`,
```

Comment:
Why this configuration has been added if there are no new serverless specific tests?

Reply:
@MadameSheema I believe this is related to this [earlier comment](https://github.com/elastic/kibana/pull/171250/files#diff-2b7a73cf2ada74d050324ff40f63d8d540e02332eae8e538b3755cc5dadaf1d1R93)

Reply:
Thanks @rylnd !! I pinged Wafaa and Diamantis since they worked together to setup serverless tests for the API framework :)